source/connect/config-settings/config-file-options.txt

```diff
@@ -0,0 +1,57 @@
+.. _config-file-options:
+
+-----------------------------------
+Compass Configuration File Settings
+-----------------------------------
+
+The following page provides an overview on how to configure settings using the 
+configuration file. For more detailed guidance on how to configure a setting, 
+see the task page for the specific setting.
+
+Definition
+----------
+
+You can configure |compass| using a configuration file. The configuration file 
+configures settings at startup. Settings configured using the configuration file 
+cannot be modified on the |compass-short| interface. 
+
+Syntax
+------
+
+You can configure |compass-short| configuration file settings in either EJSON or 
+YAML format.
+
+EJSON
+`````
+
+.. code-block:: json
+
+    { "setting": value}
+
+YAML
+````
+
+.. code-block:: yaml
+
+    setting: value
```

Comment:
[suggestion]

Maybe expand these examples out a bit to show what they look like when specifying multiple settings?

Reply:
Done.